docs/content/latest/develop/query-1-performance/explain-analyze.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+linkTitle: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+description: Query optimization with EXPLAIN and EXPLAIN ANALYZE
+aliases:
+  - /develop/query-1-performance/explain-analyze/
+headerTitle: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+image: /images/section_icons/index/develop.png
+menu:
+  latest:
+    identifier: explain-analyze
+    parent: query-1-performance
+    weight: 566
+isTocNested: true
+showAsideToc: true
+---
+
+This section describes how to optimize queries using YSQL's `EXPLAIN` and `EXPLAIN ANALYZE` statements.
+
+## The EXPLAIN Statement
+
+For each query that YSQL receives, it creates an execution plan which you can access using the `EXPLAIN` statement. This statement returns two numbers: the so-called startup cost that represents the estimated query cost before the first row is returned, and the total cost to run the query to completion.
```

Comment:
"This statement returns two numbers:" isn't the main point. That is one aspect of what it returns. But this sentence is missing to capture the main thing that EXPLAIN does or provides the user with.

Reply:
This statement was added based on a suggestion from another reviewer.

Reply:
Suggestion. Update the first sentence from:
For each query that YSQL receives, it creates an execution plan which you can access using the `EXPLAIN` statement. This statement..
--> to something like:
The EXPLAIN statement helps explain the execution plan that will be used by the database for the given statement. In addition, this statement ...